wolfssl/internal.h

```diff
@@ -3351,6 +3353,10 @@ struct WOLFSSL_SESSION {
 #endif
     byte               altSessionID[ID_LEN];
     byte               haveAltSessionID:1;
+#ifdef HAVE_EX_DATA
```

Comment:
Should this be `#if defined(HAVE_EXT_CACHE) || defined(HAVE_EX_DATA)` like the one at line 3083?

Reply:
My thinking was that we really want to store the remove session callback only when we have external data that potentially needs to be free'd that we obviously can't handle ourselves. With `HAVE_EXT_CACHE` only I don't think its necessary to store the the callback since it is called in `wolfSSL_FreeSession` whenever we are getting rid of the session.